packages/browser-wallet/src/shared/utils/verifiable-credential-helpers.ts

```diff
@@ -635,3 +649,55 @@ export async function getChangesToCredentialSchemas(
     }
     return { data: updatedSchemasInStorage, updateReceived };
 }
+
+/**
+ * Get the registry issuer public key from a credential registry CIS-4 contract.
+ * @param client the GRPC client for accessing a node
+ * @param contractAddress the address of a CIS-4 contract
+ * @returns the registry public key for the contract
+ */
+export async function getCredentialRegistryIssuerKey(
+    client: ConcordiumGRPCClient,
+    contractAddress: ContractAddress
+): Promise<string> {
+    const instanceInfo = await client.getInstanceInfo(contractAddress);
+    if (instanceInfo === undefined) {
+        throw new Error('Given contract address was not a created instance');
+    }
+
+    const result = await client.invokeContract({
+        contract: contractAddress,
+        method: `${getContractName(instanceInfo)}.issuer`,
+    });
+
+    if (result.tag !== 'success') {
+        throw new Error(result.reason.tag);
+    }
+
+    const { returnValue } = result;
+    if (returnValue === undefined) {
+        throw new Error(`Return value is missing from issuer public key result in CIS-4 contract: ${contractAddress}`);
+    }
+
+    return returnValue;
+}
+
+/**
+ * Create a publicKey DID identitifer for the given key.
+ */
+export function createPublicKeyIdentifier(publicKey: string, network: NetworkConfiguration): string {
+    return `did:ccd:${getNet(network).toLowerCase()}:pkc:${publicKey}`;
```

Comment:
Add unit tests for this and the corresponding `getPublicKeyfromPublicKeyIdentifierDID`. And for `createCredentialId`.